BallsDiscordBot.py

```diff
@@ -0,0 +1,13 @@
+import discord as Ballspy
+from Ballspy.ext import commands as BallsCommands
+
+Balls = BallsCommands.bot(command_prefix="sex")
```

Comment:
```suggestion
Balls = BallsCommands.bot(command_prefix="balls”)
```
hhaha look what i learned how to do